### ISSUE TITLE
Question about the event number loss

### ISSUE THREAD

**Issue by Tianbo-Pan** [iCVTEAM/E2NeRF#5]
Thanks for this thought-provoking work!

However, I have a question related to the event number loss.

```python
pos = event_cur > 0
neg = event_cur < 0
loss_pos = torch.mean(((thres_pos * pos) - ((event_cur + 0.5) * pos)) ** 2)
loss_neg = torch.mean(((thres_neg * neg) - ((event_cur - 0.5) * neg)) ** 2)
```
        
why the event_cur needs to +/- 0.5?

**Comment by Tianbo-Pan** [comment]
In real-world datasets, how is the camera pose determined for novel view synthesis? I couldn’t locate the relevant code for selecting these viewpoints. Could you provide clarification on this?
Thanks for your time and attention.

**Comment by 171164558** [comment]
Thanks Pan.

About the +/- 0.5. This design is used to fix the rounding operation in event generation. For example, when there is 1 positive event in the input event data. The actual number of events can be any value between 1 and 2 (If we consider the number of events as a decimal).  So, we choose 1.5 (1+0.5) as the ground truth number of events to compute the event loss, and for the negative events, it should be -1.5 (-1-0.5).

About the novel view synthesis for the real-world dataset. We use the poses for the video rendering to render the novel view images. You can just replace the "test_poses" in line 853 of run_nerf_exp.py with "render_poses" to generate the novel view images (120 images in total). We use the no-reference image quality assessment metrics to evaluate the novel view images and blur view images together (120+30=150 images).

Hope this can help you. If you have any questions, please feel free to contact me.
